test/test_distributions.py

```diff
@@ -192,3 +193,20 @@ def test_discrete_logpmf_args_check(jax_dist, dist_args):
     with pytest.raises(ValueError, match='Invalid values'):
         sample_oos = sample - 0.5
         jax_dist.logpmf(sample_oos, *dist_args)
+
+
+@pytest.mark.parametrize('jax_dist, dist_args', [
+    (dist.bernoulli, (0.1,)),
+    (dist.bernoulli, (np.array([0.3, 0.5]),)),
+], ids=idfn)
+def test_discrete_with_logits(jax_dist, dist_args):
+    rng = random.PRNGKey(0)
+    logit_args = dist_args[:-1] + (logit(dist_args[-1]),)
```

Comment:
I think you probably just meant `logit(dist_args)`?

Reply:
Yes, I mean so. I expect more distributions will need this as mentioned in https://github.com/pyro-ppl/numpyro/issues/51, so this just serves for future tests.

Reply:
@fehiepsi - I just meant why do we need to do `dist_args[:-1] + (logit(dist_args[-1]),)` instead of just using `(logit(dist_args),)`?

Reply:
For example, in binom/multinomial distributions, we have 2 args: n and p. So we only want to apply `logit` to `p`.